src/main/kotlin/org/rust/lang/core/macros/MacroExpansionTask.kt

```diff
@@ -139,7 +146,9 @@ abstract class MacroExpansionTaskBase(
                 var extractableList: List<Extractable>?
                 do {
                     extractableList = expansionSteps.nextOrNull()
-                    currentStep.incrementAndGet()
+                    val step = currentStep.incrementAndGet()
+                    MACRO_LOG.debug("Expansion step: $step")
```

Comment:
Also, let's remove this log. It's very verbose. Or maybe use TRACE level

Reply:
But why is it bothering you?
I've increased level of important messages so now you don't need to set debug level at all to see almost everything that previously required debug level